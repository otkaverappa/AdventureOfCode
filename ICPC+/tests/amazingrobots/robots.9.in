4 8
########
##.#####
......X#
########
1
2 3 2 S
7 8
########
###....#
#...####
..###..#
#..X...#
#.......
########
3
6 4 2 N
5 2 2 N
6 7 2 N
