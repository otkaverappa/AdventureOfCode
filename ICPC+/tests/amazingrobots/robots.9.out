15
E
E
S
S
E
E
E
E
W
W
W
W
W
W
W
